### PR TITLE
Updated google cloud function documentation

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description

Updated Google Cloud function documentation.
This needed an update as Google no longer supports Cloud Source Repositories for new users.

### CODE REVIEW

**Comment by dat-a-man** [comment]
@hibajamal could you please do an initial review?

File: docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-google-cloud-functions.md [hunk 1]
```diff
@@ -6,31 +6,24 @@ keywords: [how to, deploy a pipeline, Cloud Function]
 
 # Deploy a pipeline with Google Cloud Functions
 
-This guide shows you how to deploy a pipeline using the gcloud shell and `dlt` CLI commands. To deploy a pipeline using this method, you must have a working knowledge of GCP and its associated services, such as Cloud Functions, Cloud Source Repositories, Shell Editor, IAM and permissions, and GCP service accounts.
+This guide shows you how to deploy a pipeline using the gcloud shell and dlt CLI commands. To deploy a pipeline using this method, you must have a working knowledge of GCP and its associated services, such as Cloud Functions, IAM and permissions, and GCP service accounts.
```
Comment (hibajamal) [inline docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-google-cloud-functions.md:9]: are we going with dlt over `dlt`?

**Comment by hibajamal** [comment]
seems alright to me! the language fixes really made a difference

File: docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-google-cloud-functions.md (hunk 1 above)
Comment (AstrakhantsevaAA) [reply docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-google-cloud-functions.md:9]: https://www.notion.so/dlthub/Documentation-Writing-Guide-87f4fcc32655460c83bcaf7787d11e67?pvs=4#1299fb8e23cf80488adfe5fdde27907b

**Review by AstrakhantsevaAA — APPROVED** [review]
Good!

**Review by hibajamal — COMMENTED** [review]
would've approved earlier, i thought i was simply assigned (not as a reviewer) 😄

**Pull request merged**